torchgeo/datasets/cyclone.py

```diff
@@ -146,7 +146,7 @@ def _load_image(self, directory: str) -> Tensor:
                 # Moved in PIL 9.1.0
                 try:
                     resample = Image.Resampling.BILINEAR
-                except AttributeError:  # pragma: no cover
+                except AttributeError:
```

Comment:
We can finally properly test these lines!

Reply:
Finallllly!!!!!!